table.go

```diff
@@ -12,6 +12,7 @@ type Table interface {
 	Cols() map[string]string
 	Row() Row
 	Next() (bool, error)
+	Rename(old, new string) error
```

Comment:
Rather than adding a method, I suggest passing the mapping into each table constructor. This way the mapping can be applied once up front which would reduce the number of loops and what not as well. It also prevents (even for how ridiculous this may be) from calling rename in the middle of an iteration over the table.

For example, [this loop](https://github.com/chop-dbhi/diff-table/blob/f4220635af11e2bc59063f3cf2cef892e33004ef/csv.go#L63-L66) can be updated to check for a renamed column in place.